shared/chat/conversation/index.native.js

```diff
@@ -20,4 +21,11 @@ const Conversation = (props: Props) => (
   </Box>
 )
 
-export default hoc(Conversation)
+export default branch(
+  (props: Props) => !!props.rekeyInfo,
+  branch(
+    (props: Props) => props.rekeyInfo && props.rekeyInfo.get('rekeyParticipants').count(),
```

Comment:
is it possible to get into a weird state where this is zero and you can't rekey yourself on another device?